timepiece/entries/models.py

```diff
@@ -368,9 +368,17 @@ def get_total_seconds(self):
         start = self.start_time
         end = self.end_time
         if not end:
-            end = self.pause_time if self.is_paused else timezone.now()
+            if self.is_paused:
+                end = self.pause_time
+            else:
+                end = timezone.now()
         delta = end - start
-        seconds = delta.seconds - self.get_paused_seconds()
+        if self.is_paused:
+            # get_paused_seconds() takes elapsed time into account, which we do not want
+            # in this case, so subtract seconds_paused instead to account for previous pauses
+            seconds = delta.seconds - self.seconds_paused
+        else:
+            seconds = delta.seconds - self.get_paused_seconds()
```

Comment:
Thanks for looking at the PRs! This one is sneaky so I want to double check. I think the way it is written is correct. If an entry is *not* paused and active, `get_total_seconds` will get the difference between `now()` and `start`, then subtract `get_paused_seconds()`.  `get_paused_seconds()` will return `self.seconds_paused`.  

If an entry *is* paused and active, `get_total_seconds` will get the difference between `pause_time()` and `start`. It can't call `get_paused_seconds()`, because since it is paused,  `get_paused_seconds()` will include the elapsed time (thus eventually sending a negative value). So the `if self.is_paused:` is to work around `get_paused_seconds()`.